storage/pruning/pruningStorer.go

```diff
@@ -819,6 +864,36 @@ func (ps *PruningStorer) changeEpoch(header data.HeaderHandler) error {
 	return ps.removeOldPersistersIfNeeded(header)
 }
 
+func (ps *PruningStorer) createNextEpochPersisterIfNeeded(epoch uint32) {
```

Comment:
this method is very similar to this `initNextEpochPersisterIfNeeded`
are both of them needed?

Reply:
refactored to remove the duplicated code